molecule/command/init/scenario.py

```diff
@@ -37,14 +37,22 @@ class Scenario(base.Base):
 
     .. option:: molecule init scenario --scenario-name bar --role-name foo
 
-        Initialize a new scenario using a local _cookiecutter_ template. In
-        order to customise the role, please refer to the `init role` command.
+        Initialize a new scenario. In order to customise the role, please refer
+        to the `init role` command.
 
     .. program:: cd foo; molecule init scenario --scenario-name bar --role-name foo
 
     .. option:: cd foo; molecule init scenario --scenario-name bar --role-name foo
 
         Initialize an existing role with Molecule:
+
+    .. program:: cd foo; molecule init scenario --scenario-name bar --role-name foo --driver-template path
+
+    .. option:: cd foo; molecule init scenario --scenario-name bar --role-name foo --driver-template path
+
+        Initialize a new scenario using a local _cookiecutter_ template. This
```

Comment:
I don't think this describes well what this new feature is doing. It allows to customise the driver templates specifically, using the default scenario and molecule template. Right?

Reply:
Yes, how about this:
```
Initialize a new driver scenario using a local *cookiecutter* template. This allows the customization of a scenario. When the driver template is not available locally, the upstream template will be used instead.
```

Reply:
Wait, I thought it allows you to override the upstream driver folder!? What about:

> Initialize a new scenario using a local _cookiecutter_ template for the driver configuration.

I may have understood this though.

Reply:
updated

Reply:
Remove "This allows the customization of a scenario while still using the upstream driver folder."? It doesn't make sense now.